### PR TITLE
Updated VPN recommendations

### PR DESCRIPTION
**Pull request by aybatra**
# Overview/Summary

Making minor updates to recommendations for VPN

## Related Issues/Work Items

Replace this with a list of related GitHub Issues and/or ADO Work Items (Internal Only)

N/A

## This PR fixes/adds/changes/removes

1. Making minor updates to recommendations for VPN

### Breaking Changes

1. None

## As part of this Pull Request I have

- [ ] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [ ] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [ ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [x] Ensured PR tests are passing
- [ ] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [ ] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: docs/content/services/networking/vpn-gateway/_index.md [hunk 1]
```diff
@@ -66,13 +66,12 @@ To automatically deploy your virtual network gateways across availability zones,
 
 **Guidance**
 
-The active-active mode is available for all SKUs except Basic.
-Active-active gateways have two Gateway IP configurations and two public IP addresses.
+The active-active mode is available for all SKUs except Basic.You can create an Azure VPN gateway in an active-active configuration, where both instances of the gateway VMs establish S2S VPN tunnels to your on-premises VPN device. When a planned maintenance or unplanned event happens to one gateway instance, the switch over will happen automatically from the affected instance to the active instance.
```
Comment (oZakari) [inline docs/content/services/networking/vpn-gateway/_index.md:69]:
```suggestion
The active-active mode is available for all SKUs except Basic and Standard.You can create an Azure VPN gateway in an active-active configuration, where both instances of the gateway VMs establish S2S VPN tunnels to your on-premises VPN device. When a planned maintenance or unplanned event happens to one gateway instance, the switch over will happen automatically from the affected instance to the active instance.
```

image

File: docs/content/services/networking/vpn-gateway/_index.md [hunk 2]
```diff
@@ -167,7 +165,7 @@ Set up monitoring and alerts for Virtual Network Gateway health based on various
 
 **Guidance**
 
-VPN Gateway uses service health to notify about planned and unplanned maintenance. Configuring service health will notify you about changes made to your VPN connectivity.
+VPN Gateway uses service health to notify about planned and unplanned maintenance.
```
Comment (oZakari) [inline docs/content/services/networking/vpn-gateway/_index.md:168]:
```suggestion
VPN Gateway uses service health alerts to notify about planned and unplanned maintenance.
```

Comment (oZakari) [reply docs/content/services/networking/vpn-gateway/_index.md:168]: I think the recommendation name should also be changed to `Enable Service Health Alerts`?

**Review by oZakari — CHANGES_REQUESTED** [review]

**Review by oZakari — CHANGES_REQUESTED** [review]
Can you please also update the PR form and fix the lint errors (bottom of PR)?

File: docs/content/services/networking/vpn-gateway/_index.md (hunk 1 above)
Comment (aybatra) [reply docs/content/services/networking/vpn-gateway/_index.md:69]: @oZakari Standard SKU is on the deprecation path and is announced to be deprecated in 2 years. No new standard SKUs can be created starting Dec 2023. There is no guidance on Standard SKU on our documentation so we do not want to add that.

File: docs/content/services/networking/vpn-gateway/_index.md (hunk 2 above)
Comment (aybatra) [reply docs/content/services/networking/vpn-gateway/_index.md]: These recommendations look good, updated this @oZakari

File: docs/content/services/networking/vpn-gateway/_index.md (hunk 1 above)
Comment (oZakari) [inline docs/content/services/networking/vpn-gateway/_index.md:69]:
```suggestion
The active-active mode is available for all SKUs except Basic. You can create an Azure VPN gateway in an active-active configuration, where both instances of the gateway VMs establish S2S VPN tunnels to your on-premises VPN device. When a planned maintenance or unplanned event happens to one gateway instance, the switch over will happen automatically from the affected instance to the active instance.
```

**Review by oZakari — CHANGES_REQUESTED** [review]

**Review by oZakari — APPROVED** [review]

**Pull request merged**